airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionManagerWorkflowImpl.java

```diff
@@ -398,4 +329,141 @@ private void continueAsNew(final ConnectionUpdaterInput connectionUpdaterInput)
     }, input);
   }
 
+
+  /**
+   * Return the duration to wait. This is calculated by the configFetchActivity and return the
```

Comment:
can we explain here why we need to wait?

Reply:
Updated.

Reply:
Gotcha. So this is wait function is how we 'do scheduling'?  I added a suggestion to make that clearer.

Reply:
How does the `configFetchActivity` schedule? It looks like since the previous run start? Is that right? Can we add these details to the getTimeToWait function doc string?

Reply:
> Gotcha. So this is wait function is how we 'do scheduling'? I added a suggestion to make that clearer.

Yes it is.

Reply:
> How does the `configFetchActivity` schedule? It looks like since the previous run start? Is that right? Can we add these details to the getTimeToWait function doc string?

Done